test/e2e/spec/main_spec.js

```diff
@@ -1,24 +1,22 @@
 describe("Ngnix/django", function() {
 
-	var frisby = require('frisby');
+	var chakram = require('/usr/local/lib/node_modules/chakram/lib/chakram.js'),
+	expect = chakram.expect;
```

Comment:
don't make `expect` global, i.e. use `var expect = ...`

Reply:
I just realized there is a `,` ending the previous line; please fix indentation to make clear this is still belonging to the `var` statement.